apps/views.py

```diff
@@ -55,38 +56,33 @@ def index(request, user, project):
 
 @permission_required_or_403("can_view_project", (Project, "slug", "project"))
 def logs(request, user, project, ai_id):
-    template = "logs.html"
+    template = "apps/logs.html"
     app = AppInstance.objects.get(pk=ai_id)
     project = Project.objects.get(slug=project)
     app_settings = app.app.settings
-    containers = []
     logs = []
+    # Look for logs in app settings. TODO: this logs entry is not used. Remove or change this later.
     if "logs" in app_settings:
-        containers = app_settings["logs"]
-        container = containers[0]
-        print("default container: " + container)
-        if "container" in request.GET:
-            container = request.GET.get("container")
-            print("Got container in request: " + container)
-
         try:
             url = settings.LOKI_SVC + "/loki/api/v1/query_range"
             app_params = app.parameters
-            print('{container="' + container + '",release="' + app_params["release"] + '"}')
+            print('{release="' + app_params["release"] + '"}')
             query = {
-                "query": '{container="' + container + '",release="' + app_params["release"] + '"}',
-                "limit": 50,
-                "start": 0,
+                "query": '{release="' + app_params["release"] + '"}',
+                "limit": 500,
```

Comment:
I guess we don't offer pagination right now, right?

Reply:
I have added datatables now which offers pagination by default.